README.md

```diff
@@ -40,3 +65,68 @@ export NVM_DIR="$HOME/.nvm"
 **module not found**
 
 If "module not found" errors are encountered when running `yarn test` or from husky precommit checks, then please make sure packages are built first by running `yarn dev` or `yarn build` from root directory.
+
+## Contributing
+
+<a href="https://github.com/DistributedCollective/sovryn-dapp/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=DistributedCollective/sovryn-dapp" />
+</a>
+
+### Support Questions
+
+Sovryn's GitHub issue trackers are not intended to provide help or support. Use one of the following channels instead:
+
+- [Discord](https://discord.gg/kBTNx4zjRf)
+- [Wiki Pages](https://wiki.sovryn.app)
+- [Sovryn Forum](https://forum.sovryn.app)
+- [Sovryn Blog](https://sovryn.com/all-things-sovryn)
+
+### Bug Reports
+
+To foster active collaboration, Sovryn strongly encourages the creation of pull requests rather than just bug reports. "Bug reports" may also be sent in the form of a pull request containing a failing test.
+
+However, if you file a bug report, your issue should contain a title and a clear description of the issue. You should also include as much relevant information as possible. The goal of a bug report is to make it easy for yourself - and others - to replicate the bug and develop a fix.
+
+Remember, bug reports are created in the hope that others with the same problem will be able to collaborate with you on solving it. Do not expect that the bug report will automatically see any activity or that others will jump to fix it. Creating a bug report serves to help yourself and others start on the path of fixing the problem. If you want to chip in, you can help out by fixing any bugs listed in our [issue trackers](https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Abug+user%3Adistributedcollective).
+
+### Which Branch?
+
+The `develop` branch acts as a testnet containing the latest changes. The `main` branch is production branch for the **app.sovryn.com**. Depending on your feature you are contributing with, select the proper branch as a starting point. Most of the time, it will be the `develop` branch unless you provide hotfixes or features that should be released before other features - then it can be `main`. By doing so, we merge all features to `develop` and then `develop` to `main` to make one big release batch, after full testing and review.
```

Comment:
```suggestion
The `develop` branch acts as a testnet containing the latest changes. The `main` branch is production branch for the **sovryn.app**. Depending on your feature you are contributing with, select the proper branch as a starting point. Most of the time, it will be the `develop` branch unless you provide hotfixes or features that should be released before other features - then it can be `main`. By doing so, we merge all features to `develop` and then `develop` to `main` to make one big release batch, after full testing and review.
```
I think it should be `sovryn.app`.